src/grapheme.rs

```diff
@@ -482,6 +488,7 @@ impl GraphemeCursor {
         self.state = GraphemeState::Emoji;
     }
 
+    #[inline]
```

Comment:
this should probably go below the docs

Reply:
Thanks. I thought I saw this pattern somewhere else in the crate and decided to follow. Will submit a follow-up PR that moves all of them below the docs.

Reply:
sounds good!